node/node.go

```diff
@@ -133,11 +133,8 @@ func New(conf *Config) (*Node, error) {
 	node.server.Config.PrivateKey = node.config.NodeKey()
 	node.server.Config.Name = node.config.NodeName()
 	node.server.Config.Logger = node.log
-	if node.server.Config.StaticNodes == nil {
-		node.server.Config.StaticNodes = node.config.StaticNodes()
-	}
-	if node.server.Config.TrustedNodes == nil {
-		node.server.Config.TrustedNodes = node.config.TrustedNodes()
+	if node.server.Config.StaticNodes == nil || node.server.Config.TrustedNodes == nil {
+		node.config.checkLegacyFiles()
 	}
```

Comment:
```suggestion
	node.config.checkLegacyFiles()
```
It's a pretty cheap check... Might help poke people to remember to clean it up. Also, just doing a `geth .. dumpconfig` spits out 
```
$ cat conf.toml| grep "icNodes\|edNodes"
StaticNodes = []
TrustedNodes = []
```
I.E: empty but _non-nil_. So a lot of default configs have these empty lists and would therefore silently avoid the legacy file-check.